dune-project

```diff
@@ -21,8 +21,9 @@
   (dune (>= 2.3))
   (emile (>= 1.0))
   (mrmime (>= 0.3.1))
-  (colombe (>= 0.3.0))
-  (sendmail-lwt (>= 0.3.0))
+  (colombe (>= 0.4.0))
+  (sendmail (>= 0.4.0))
```

Comment:
Why was this added?

Reply:
I've fixed now all dependencies to be correct. Correct dependency is really to have `sendmail`, because this code does not use `sendmail-lwt`. Code should be refactored to remove `sendmail_handler.*` files and `sendmail-lwt` should be used directly.

Reply:
Ah, was it a transitive dependency before?